troposphere/static/js/models/Instance.js

```diff
@@ -2,11 +2,12 @@ define(
   [
     'backbone',
     'underscore',
+    'jquery',
     'globals',
     'context',
     './InstanceState'
   ],
-  function (Backbone, _, globals, context, InstanceState) {
+  function (Backbone, _, $, globals, context, InstanceState) {
```

Comment:
Ahh gotchya. Thanks for confirmation and clarification that jquery is now explicitly required for components that use it now that we're on webpack.